.travis.yml

```diff
@@ -53,12 +45,13 @@ env:
 before_script:
   - sudo iptables -L DOCKER || ( echo "DOCKER iptables chain missing" ; sudo iptables -N DOCKER )
   - eval "$(/opt/chefdk/bin/chef shell-init bash)"
+  - chef gem install kitchen-dokken
```

Comment:
yeah but we needed an updated version, as ChefDk released a few days before some container fixes went in.